marklogic-data-hub/src/test/java/com/marklogic/hub/impl/ClearUserModulesTest.java

```diff
@@ -21,6 +22,11 @@
 
 class ClearUserModulesTest extends AbstractHubCoreTest {
 
+    @BeforeEach
+    public void clean() {
+        new DataHubImpl(getHubConfig()).clearUserModules();
```

Comment:
Do "clearUserModules()" instead - that will avoid deleting the marklogic-unit-test REST extension. That method should be used throughout this test.